iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -159,11 +160,42 @@ <h3 class="collapsible-header" data-i18n="inspector.associatedFacts">Associated
   </div>
   <div class="dialog-mask">
   </div>
-  <div id="chart" class="dialog">
-    <div class="close"></div>
-    <div class="contents">
-      <div class="chart-container" ><canvas id='chart-canvas'></canvas></div>
-      <div class="other-aspects"></div>
+  <div id="dialog-container"></div>
+  <div id="dialog-templates">
+    <div class="dialog full-screen chart">
+      <div class="close"></div>
+      <div class="contents">
+        <div class="chart-container" ><canvas id='chart-canvas'></canvas></div>
+        <div class="other-aspects"></div>
+      </div>
+    </div>
+    <div class="dialog validation-report">
+      <div class="close"></div>
+      <div class="contents">
+          <h2>Validation results</h2>
```

Comment:
Does this label (and the th below) need i18n?

Reply:
Good point - fixed in cf5c949